main.cpp

```diff
@@ -61,19 +61,48 @@ int main()
 	*/
 
 
-	
-	string pathName = "F:/FishExpData/";
-	ExperimentData exp(pathName);
 
-	if (!exp.initialize())
+	Timer expTimer;
+	ScreenData Board;
+	vector<string> CSpattern;
+	CSpattern.push_back("Images/pureBlack.jpg");
+
+	Board.initialize(CSpattern);
```

Comment:
better to encapsulate line 70 - line 82 into a method of ExperimentData Class

Reply:
Sure, I will do it latter

Reply:
I just reconsidered it. This part is just to test whether the modification of ScreenData, AreaData and PatchData correct. It may be a method in the ScreenData class in the future. I will encapsulate this part of code into a method of ScreenData while writing the new method which carries out the whole experiment.